deploy/cloud_providers/aws/restapi/roles/templates/role_policy.source.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::754728514883:root"
```

Comment:
should the account id be override-able / interpolated?

Reply:
Yeah I scratched my head on that one, but it's actually a NewRelic account id that's going to pull the data, so this authorizes us (newrelic) to get that access.
![image](https://user-images.githubusercontent.com/38018054/96935764-98145600-1479-11eb-9f41-ce78f3afccc9.png)

Reply:
consider this resolved then.